src/ape_test/_cli.py

```diff
@@ -0,0 +1,23 @@
+import sys
+
+import click
+import pytest
+
+from ape.cli import network_option
+
+
+@click.command(
+    add_help_option=False,  # NOTE: This allows pass-through to pytest's help
+    short_help="Launches pytest and runs the tests for a project",
+    context_settings=dict(ignore_unknown_options=True),
+)
+@network_option()
+@click.argument("pytest_args", nargs=-1, type=click.UNPROCESSED)
+def cli(network, pytest_args):
+    if "--network" not in pytest_args:
+        pytest_args = [*pytest_args, "--network", network]
```

Comment:
Is `pytest_args` overriding the network option?

Reply:
If you do `ape test --network <network>`, it gets added to pytest_args.
If you do `ape --network <network> test`, it gets added to the network option.
If you do the latter, we just inject it to the pytest args. This way, we don't start up any providers unless tests are collected

Reply:
I guess `ape --network <network> test` does not actually work so this is not really needed, I just removed!